src/components/ComputeModal.js

```diff
@@ -802,7 +808,10 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
         ...commonButtonProps,
         onClick: () => {
           applyChanges()
-        }
+        },
+        disabled: isUpdateDisabled,
+        tooltipSide: 'left',
+        tooltip: isUpdateDisabled ? `Cannot perform change on environment in ( ${currentRuntimeDetails.status} ) status` : 'Update Environment'
```

Comment:
I'm sure UX didn't want it to look like this. I think the parens need to go, but you can certainly run this by them!

![image](https://user-images.githubusercontent.com/484484/161168295-c7dd6160-c01d-4c24-acc7-1237ff9dec7c.png)

Reply:
Ah, it looks better in the tooltip from the context bar. I'll remove the spaces here.